lib/eslint.js

```diff
@@ -166,6 +168,14 @@ function addDeclaredGlobals(program, globalScope, config) {
         }
         variable.writeable = explicitGlobals[name];
     });
+
+    // mark all exported variables as such for no-unused-var
+    Object.keys(exportedGlobals).forEach(function(name) {
+        var variable = getVariable(globalScope, name);
```

Comment:
I though that Node does have global scope, it's just not easy to get to. I'm I wrong about that?